src/App.tsx

```diff
@@ -54,11 +54,17 @@ import NurseryDetails from './components/Nursery';
 import InventoryCreate from './components/Inventory/InventoryCreate';
 import InventoryView from './components/Inventory/InventoryView';
 
+interface StyleProps {
+  isDesktop?: boolean;
+}
+
 const useStyles = makeStyles((theme: Theme) => ({
   container: {
     '& .navbar': {
       backgroundColor: 'transparent',
-      paddingTop: '64px',
+      paddingTop: (props: StyleProps) => (props.isDesktop ? '64px' : '8px'),
+      overflowY: 'auto',
```

Comment:
for some reason this is not working in my vercel view, the scroll is happening on the background content.

![screencast 2022-11-07 11-13-57](https://user-images.githubusercontent.com/1865174/200395502-ecaf080f-e032-42d8-9383-0be503b907f8.gif)

Reply:
this is on chrome

Reply:
You can't see the "Contact us"? It looks like your windows is bigger enough to show all the items because scroll bar is not there. Can you try making the window smaller in height?  I'm also using chrome and the scroll bar is appearing

Reply:
Same issue even with a very small height. Were you able to see scrolling with the vercel link?
Maybe something at my end.